pyscriptjs/tests/integration/test_02_display.py

```diff
@@ -219,6 +219,31 @@ def test_display_multiple_append_false(self):
         pattern = r'<py-script id="py-.*">world</py-script>'
         assert re.search(pattern, inner_html)
 
+    def test_display_multiple_append_false_with_target(self):
+        self.pyscript_run(
+            """
+            <div id="circle-div"></div>
+            <py-script>
+                class Circle:
+                    r = 0
+                    def _repr_svg_(self):
+                        return f'<svg height="{self.r*2}" width="{self.r*2}"><circle cx="{self.r}" cy="{self.r}" r="{self.r}" fill="red" /></svg>' # noqa: E501
+
+                circle = Circle()
+
+                circle.r += 5
+                display(circle, target="circle-div", append=False)
+                circle.r += 5
+                display(circle, target="circle-div", append=False)
+            </py-script>
+        """
+        )
+        innerhtml = self.page.locator("id=circle-div").inner_html()
+        assert (
+            innerhtml
+            == '<svg height="20" width="20"><circle cx="10" cy="10" r="10" fill="red"></circle></svg>'  # noqa: E501
+        )
```

Comment:
Hey @madhur-tandon thank you for implementing a fix on this and adding a test!
I think `<svg height="20" width="20"><circle cx="10" cy="10" r="10" fill="red"></circle></svg>` this should be the right answer once you click the button, according to the issue, right?
So I guess we should also implement a button here and a click... What do you think?
(If you're basing yourself in the example).

Reply:
I can implement a button click but then, I just called `display` twice...
The original issue is not about a button click, but calling `display` twice.

Reply:
Oh, sorry I missed you called it twice.